patches/chrome-browser-ui-views-page_info-page_info_cookies_content_view.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc b/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc
+index 013518134ff3f4d4fb09bab686da9457c2c0e72d..665c543c8133d98b3b2a822fe9e307ec0f81f5ca 100644
+--- a/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc
++++ b/chrome/browser/ui/views/page_info/page_info_cookies_content_view.cc
+@@ -422,7 +422,7 @@ void PageInfoCookiesContentView::SetThirdPartyCookiesInfo(
+     CookieControlsEnforcement enforcement,
+     CookieBlocking3pcdStatus blocking_status,
+     base::Time expiration) {
+-  if (controls_state == CookieControlsState::kHidden) {
++  if (controls_state == CookieControlsState::kHidden || true) { // Always bail out as we hide cookie descriptions
```

Comment:
Can this be done with an override of `kHidden` or something similar?

Reply:
Yeah, it could cause more problems down the line, good point. But then I guess we'd want to at least make this patch extensible via a `chromium_src` definition, right?

Reply:
Can we make SetThirdPartyCookiesInfo virtual and subclass + override?

Reply:
Done.